
SOMObservation x
Relation relation
SOMObservation y
geometry_msgs/Pose current_robot_pose


---

Match[] matches
